gradle/intTestImage.gradle

```diff
@@ -38,12 +39,46 @@ if (useAllDistribution) {
         from(unpackedPath)
     }
 } else {
+    configurations { selfRuntime }
+    if (project.tasks.findByName("jar")) {
```

Comment:
Why not just add the jar directly?

Reply:
I couldn't figure out how to do that in a lazy way. Such that _config1 + config2_  further down still works. We need this to figure out if the `jar` should go into `lib` or `lib/plugins`.

I first wanted to do this:

```
into('lib') { // <- or lib/plugins ?
   from jar
}
```

A solution could be to improve how we define what a _plugin_ is so that we have a complete list of projects that go into `plugins`. But still then we would do the _Is this a plugin?_-check twice. Once for the configurations and once for the jar.